packages/sdk/src/domain/services/TokenConversionService.ts

```diff
@@ -0,0 +1,45 @@
+import { Currency } from "@heliofi/common";
+import type { CurrencyService } from "./CurrencyService";
+
+export class TokenConversionService {
+  // @TODO change to bigint
```

Comment:
is it a lot of work to change everything to bigint already? we are currently doing this to dash

Reply:
hmmm. the problem is that in react it's not done. so I think we can do that with react refactoring. do you agree?